tests/gradients/test_param_shift_hessian.py

```diff
@@ -80,7 +80,7 @@ def test_single_two_term_gate(self):
 
         dev = qml.device("default.qubit", wires=2)
 
-        @qml.qnode(dev, diff_method="parameter-shift", max_diff=2)
```

Comment:
How come this was deleted? Without this being specified, the default would switch to backprop

Reply:
Ah, yes, we want that I think, because the internal QNode logic is only used to compute `expected`, and backprop shold be faster, right? `param_shift_hessian` is called explicitly, so it still tests that function as intended.

Reply:
You're right... but it still makes me nervous to change what the existing test suite is testing 😆 

In an ideal world it doesn't matter whether the tests are backprop or param-shift, but I wonder if there are edge cases that arise using the hessian transform using one rather than the other.

I would recommend either:

- Reverting this change, or
- Maybe we should actually parametrize this test class over both backprop and parameter-shift?

Reply:
I checked, and the difference in execution time is not too major, so I'll revert the change.